### PR TITLE
Document constructor arguments

### LINKED ISSUE #1209: Document constructor arguments of ServiceRegistry
As @karlb [pointed out](https://github.com/raiden-network/raiden-contracts/pull/1207#pullrequestreview-282458735), many constructor arguments of ServiceRegistry are not documented in the source.  This issue keeps track of describing the constructor arguments.

### PR DESCRIPTION
**Pull request by pirapira**
This fixes #1209.

### What this PR does

This commit adds parameter docstrings to smart contracts in
ServiceRegistry.sol.

### Why I'm making this PR

Only some of the parameters were documented.

### What's tricky about this PR (if any)

Nothing.

----

Any reviewer can check these:

* [ ] If the PR is fixing a bug or adding a feature, add an entry to CHANGELOG.md.
* [ ] If the PR changed a Solidity source, run `make compile_contracts` and add the resulting `raiden_contracts/data/contracts.json` in the PR.
* [ ] If the PR is changing documentation only, add `[skip ci]` in the commit message so Travis does not waste time.
    * [ ] But, if the PR changes comments in a Solidity source, do not add `[skip ci]` and let Travis check the hash of the source.
* [ ] In Python, use keyword arguments
* [ ] Squash unnecessary commits
* [ ] Comment commits
* [ ] Follow naming conventions
    * `solidityFunction`
    * `_solidity_argument`
    * `solidity_variable`
    * `python_variable`
    * `PYTHON_CONSTANT`
* [ ] Follow the Signature Convention in [CONTRIBUTING.md](./CONTRIBUTING.md)
* For each new contract
    * [ ] The deployment script deploys the new contract.
    * [ ] `etherscan_verify.py` runs on the new contract.
* Bookkeep
    * [ ] The gas cost of new functions are stored in `gas.json`.
* Solidity specific conventions
    * [ ] Document arguments of functions in natspec
    * [ ] Care reentrancy problems
* [ ] When you catch a require() failure in Solidity, look for a specific error message like `pytest.raises(TransactionFailed, match="error message"):`

And before "merge" all checkboxes have to be checked.  If you find redundant points, remove them.

### CODE REVIEW

File: raiden_contracts/data/source/services/ServiceRegistry.sol [hunk 1]
```diff
@@ -220,6 +221,13 @@ contract ServiceRegistry is Utils, ServiceRegistryConfigurableParameters {
     event RegisteredService(address indexed service, uint256 valid_till, uint256 deposit_amount, Deposit deposit_contract);
 
     // @param _token_for_registration The address of the ERC20 token contract that services use for registration fees
+    // @param _controller The address that can change parameters and deprecate the ServiceRegistry
```
Comment (palango) [inline raiden_contracts/data/source/services/ServiceRegistry.sol]: Some of them end with a dot, some don't

File: raiden_contracts/data/source/services/ServiceRegistry.sol [hunk 2]
```diff
@@ -220,6 +221,13 @@ contract ServiceRegistry is Utils, ServiceRegistryConfigurableParameters {
...
+    // @param _initial_price The amount of tokens needed iniailly for a slot
```
Comment (palango) [inline raiden_contracts/data/source/services/ServiceRegistry.sol]: s/iniailly/initially

File: raiden_contracts/data/source/services/ServiceRegistry.sol [hunk 3]
```diff
@@ -220,6 +221,13 @@ contract ServiceRegistry is Utils, ServiceRegistryConfigurableParameters {
...
+    // @param _price_bump_numerator The ratio of price bump after deposit is made (numerator)
+    // @param _price_bump_denominator The ratio of price bump after deposit is made (denominator)
+    // @param _decay_constant The number of seconds after which the price becomes roughly 1/e.
+    // @param _min_price The minimum amount of tokens needed for a slot.
+    // @param _registration_duration The number of seconds (roughly) of a slot gained for a successful deposit.
```
Comment (palango) [inline raiden_contracts/data/source/services/ServiceRegistry.sol]: Is this roughly defined somewhere?

Comment (pirapira) [reply raiden_contracts/data/source/services/ServiceRegistry.sol]: I'll change "roughly" into "roughly, barring block time & miners' timestamp errors".

File: raiden_contracts/data/source/services/ServiceRegistry.sol (hunk 1 above)
Comment (pirapira) [reply raiden_contracts/data/source/services/ServiceRegistry.sol]: Done.

File: raiden_contracts/data/source/services/ServiceRegistry.sol (hunk 3 above)
Comment (pirapira) [reply raiden_contracts/data/source/services/ServiceRegistry.sol]: Done.

File: raiden_contracts/data/source/services/ServiceRegistry.sol (hunk 2 above)
Comment (pirapira) [reply raiden_contracts/data/source/services/ServiceRegistry.sol]: Fixed.

**Pull request merged**